testdata/p4_14_samples_outputs/01-BigMatch-first.p4

```diff
@@ -51,7 +51,7 @@ control egress(inout headers hdr, inout metadata meta, inout standard_metadata_t
             @defaultonly NoAction();
         }
         key = {
-            hdr.ethernet.srcAddr: exact @name("hdr.ethernet.srcAddr") ;
+            hdr.ethernet.srcAddr: exact @name("ethernet.srcAddr") ;
```

Comment:
is this what you expect?
what happens to 'hdr'? How does the control-plane make up that?

Reply:
Yeah, that change was one of the main things I was trying to accomplish in this PR. `hdr` shouldn't be visible to the control plane, because it's not part of the P4-14 program; it's just an artifact of the conversion from P4-14 to P4-16. That was already true, because the P4Runtime code ensured that, but now that I'm making the P4Runtime code lean more heavily on the frontend code, I need to make the appropriate changes to maintain that behavior. Otherwise, the control plane API we'll generate will be incompatible with the one we generate from the old compiler.